app/views/admin/layouts/edit.html.erb

```diff
@@ -6,8 +6,9 @@
   </div>
   <div class="row">
     <div class="layout-area col-sm-12">
-        <%= render partial: 'area', collection: @desks %>
-        <%= render partial: 'area', collection: @child_groupings %>
+      <% @children.each do |child| %>
+      <%= area_tag(child, ['draggable', child.type]) %>
```

Comment:
If you make use of variable number of arguments in `area_tag` you'll be able to just do:
```ruby
area_tag(child, 'draggable', child.type)
```
  
Also, you missed an indent inside the loop 😉